ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+[//]: # (DELETE THIS LINE: Choose one of the following two issue types and delete the other.)
+
+### Document Issue
+
+**Document Location**: [PATH_TO_DOCUMENT](FULL_LINK)
+
+[//]: # (DELETE THIS LINE: If the document exists, replace PATH_TO_DOCUMENT with the path to the document header for example, path/name.md#header and replace FULL_LINK with full link of the path to the document header.. If you are proposing a new document, replace PATH_TO_DOCUMENT with the path to the document header of where it should go.)
```

Comment:
Please replace path/name.md#header with a path to a real document header within this repo